services/src/test/scala/cromwell/pipeline/service/impls/ProjectFileServiceTestImpl.scala

```diff
@@ -7,7 +7,8 @@ import cromwell.pipeline.service.{ ProjectFileService, VersioningException }
 import java.nio.file.Path
 import scala.concurrent.Future
 
-class ProjectFileServiceTestImpl(projectFiles: Seq[ProjectFile], testMode: TestMode) extends ProjectFileService {
+class ProjectFileServiceTestImpl(projectFileBundles: Seq[(ProjectId, Seq[ProjectFile])], testMode: TestMode)
```

Comment:
Sorry for not saying about it earlier, but you should definitely use `Map[ProjectId, Seq[ProjectFile]]` here

Reply:
Fixed, looks prettier and more simple indeed